R/PipeOpFFS.R

```diff
@@ -57,10 +59,40 @@ PipeOpFFS = R6Class("PipeOpFFS",
         drop = p_lgl(tags = c("train", "predict", "required")),
         left = p_dbl(tags = c("train", "predict", "required")),
         right = p_dbl(tags = c("train", "predict", "required")),
-        feature = p_fct(
-          levels = c("mean", "max", "min", "slope", "median", "var"),
-          tags = c("train", "predict", "required")
-        )
+        features = p_uty(tags = c("train", "predict", "required"), custom_check = function(x) {
+          if (!(is.character(x) || is.list(x))) {
```

Comment:
For that we often use checkmate functions like `test_character()` or `test_list()`

Reply:
(similar for the other functions). But if you prefer the base R functions I am fine with that too :)

Reply:
changed it, but isn't there unnecessary overhead for just checking the class?

Reply:
the underlying checkmate function is written in C as well